src/components/UserInfo.js

```diff
@@ -1,15 +1,38 @@
+import Api from "./Api.js";
+
 export default class UserInfo {
-  constructor(nameElement, titleElement) {
-    this._name = nameElement;
-    this._title = titleElement;
+  constructor({ name, about, avatar, apiHeaders, apiUser }) {
+    this.name = name;
+    this.about = about;
+    this.avatar = avatar;
+    this._headers = apiHeaders;
+    this._endpoint = apiUser;
   }
 
   getUserInfo() {
-    return { name: this._name.textContent, title: this._title.textContent };
```

Comment:
`   return { name: this._name.textContent, title: this._title.textContent };` should be here like it was previously


You don't need to waste resources on fetching the data when you open the profile form